tests/topotests/isis_srv6_topo1/test_isis_srv6_topo1.py

```diff
@@ -238,17 +238,34 @@ def router_compare_json_output(rname, command, reference):
 
 
 def check_ping6(name, dest_addr, expect_connected):
```

Comment:
do we have multiple ping6 helper apis like this one? should we put a single version into lib/ ?

Reply:
I thought about it but there are like 100 bajillion places that ping would need to be touched and I didn't want to take that on today.   I just wanted this test to pass.  Yes I realize bad donald.  but I don't particularly want to be programming in python to dig ourselves out of this stupid hole.